build/build.xml

```diff
@@ -48,6 +49,12 @@ External Dependencies:
 	<echo>Loading base properties</echo>
 	<loadproperties srcFile="build.properties"/>
 
+	<!-- Load secret properties -->

```

Comment:
Those aren't new properties.  They were just moved to a different place in the build file.  They're used for the secure signing of our deb build